LegitimateURLShortener.txt

```diff
@@ -1950,6 +1950,9 @@ $removeparam=fromRankId
 ||bilibili.com^$removeparam=refer_from
 ! https://show.bilibili.com/platform/home.html?msource=pc_web
 ||bilibili.com^$removeparam=msource
+! https://github.com/the1812/Bilibili-Evolved/issues/2247
+! This script is used to add the spm_id_from parameter to the hyperlink on the web page.
+||s1.hdslb.com/bfs/seed/jinkela/short/auto-append-spmid.js
```

Comment:
I do not think this is really the purpose of Legit URL Shortener, but I'd have to ask Imre.
@DandelionSprout your opinion?

Reply:
After thinking about it a bit, I think there are grounds to shorten it to simply `/auto-append-spmid.js` even.